addons/website/models/website.py

```diff
@@ -479,6 +479,10 @@ def set_images(images):
         theme_name = kwargs['theme_name']
         theme = self.env['ir.module.module'].search([('name', '=', theme_name)])
         url = theme.button_choose_theme()
+        # Redirect to the default language of the website
+        # otherwise the user could get stuck in a loop
```

Comment:
It is not really a loop, the comment is misleading. It is just the loading animation never being removed as the code is never catching the `enable_editor=1` correctly as there is no Edit button in translation mode, thus the Edit Mode is never started